.gitlab/jobs/lassen.yml

```diff
@@ -16,10 +16,10 @@
 # Overriding shared spec: Longer allocation + extra flags
 xl_2022_08_19_gcc_8_3_1_cuda_11_2_0:
   variables:
-    SPEC: "${PROJECT_LASSEN_VARIANTS} +cuda cxxflags==\"-qthreaded -std=c++14 -O3 -qstrict -qxlcompatmacros -qlanglvl=extended0x -qalias=noansi -qhot -qpic -qsmp=omp -qsuppress=1500-029 -qsuppress=1500-036\" %xl@16.1.1.12.gcc.8.3.1 ^cuda@11.2.0+allow-unsupported-compilers ${PROJECT_LASSEN_DEPS}"
+    SPEC: "${PROJECT_LASSEN_VARIANTS} +cuda cxxflags==\"-qthreaded -std=c++14 -O3 -qstrict -qxlcompatmacros -qlanglvl=extended0x -qalias=noansi -qhot -qpic -qsmp=omp -qsuppress=1500-029 -qsuppress=1500-036\" %xl@16.1.1.12.gcc.8.3.1 ^cuda@11.2.0+allow-unsupported-compilers ${PROJECT_LASSEN_DEPS} ^blt@git.a7f0a6ecc4fdfa1724399b1454c3909b9ee02e81=develop"
```

Comment:
Per .gitlab/custom_jobs_and_variables the variable PROJECT_LASSEN_DEPS is already set to: "^blt@git.a7f0a6ecc4fdfa1724399b1454c3909b9ee02e81=develop"

Reply:
I was following what is currently done in RAJA.  I will make the change when CI passes. Please let me know if I see any other issues with bad form, etc.

Reply:
@adrienbernede I think I've addressed your comments about applying the proper spec conventions in overridden vs. extra jobs.